azure-kusto-ingest/azure/kusto/ingest/_ingestion_properties.py

```diff
@@ -135,5 +155,7 @@ def get_mapping_format(self):
             return DataFormat.json.name
         elif self.format == DataFormat.avro:
             return DataFormat.avro.name
+        elif self.format == DataFormat.parquet:
```

Comment:
You can remove this method and use the new property "ingestionMappingReference"

Reply:
This method is also used with the property `mapping`. I'll see if I can remove it as well